### PR TITLE
Update AltV version , Clean up, Move refs 

### PR DESCRIPTION
**Pull request by Kaiwoknats**

### CODE REVIEW

File: AltV.Atlas.Boilerplate.Server/Startup.cs [hunk 1]
```diff
@@ -8,9 +8,9 @@ namespace AltV.Atlas.Boilerplate.Server;
 
 public class Startup : AsyncResource
 {
+    private static void Main( string[] args ){}
```
Comment (TheDutchDev) [inline AltV.Atlas.Boilerplate.Server/Startup.cs:11]: Why?

File: AltV.Atlas.Boilerplate.Server/Startup.cs [hunk 2]
```diff
@@ -30,4 +30,4 @@ public override IEntityFactory<IVehicle> GetVehicleFactory( )
     {
         return _bootstrapper.Value.Services.GetService<IEntityFactory<IAtlasVehicle>>( )!;
     }
-}
\ No newline at end of file
+}
```
Comment (TheDutchDev) [inline AltV.Atlas.Boilerplate.Server/Startup.cs:33]: same for this one

Comment (Kaiwoknats) [reply AltV.Atlas.Boilerplate.Server/Startup.cs:33]: git bug idk didnt change anything

File: AltV.Atlas.Boilerplate.Client/Bootstrapper.cs [hunk 3]
```diff
@@ -30,20 +33,34 @@ private void ConfigureServices( )
...
+
         #endregion
     }
 
     public void Run( )
     {
+        #region Free Modules
+
+        _serviceProvider.InitializeScaleformModule( );
+        _serviceProvider.InitializeKeyInputModule( new List<Key>{Key.Space, Key.F4} );
```
Comment (TheDutchDev) [inline AltV.Atlas.Boilerplate.Client/Bootstrapper.cs]:
```suggestion
       var whitelistedKeys = new List<Key>{Key.Space, Key.F4};
        _serviceProvider.InitializeKeyInputModule( whitelistedKeys );
```

makes it more clear what that piece of code is doing

File: AltV.Atlas.Boilerplate.Client/appsettings.json [hunk 4]
```diff
@@ -1,4 +1,12 @@
 {
+  "TrafficSettings": {
+    "MaxTrafficVehiclesInStreamDistance": 30,
+    "MinimumTrafficNodesToGenerate": 10,
+    "SpawnRadius": 160,
+    "MinimumSpawnDistanceFromPlayer": 80,
+    "CleanupIntervalMs": 60000,
+    "TrafficSpawnIntervalMs": 1000
+  },
```
Comment (TheDutchDev) [inline AltV.Atlas.Boilerplate.Client/appsettings.json]:
```suggestion
```

As this is part of a premium module, it doesn't need to be in appsettings by default.

File: AltV.Atlas.Boilerplate.Server/appsettings.json [hunk 5]
```diff
@@ -1,4 +1,15 @@
...
+    "MaxTrafficVehiclesInStreamDistance": 30,
+    "MinimumTrafficNodesToGenerate": 10,
+    "SpawnRadius": 160,
+    "MinimumSpawnDistanceFromPlayer": 80,
+    "CleanupIntervalMs": 60000,
+    "TrafficSpawnIntervalMs": 1000
+  },
+  "Database": {
+    "ConnectionString": "Server=localhost;Database=atlas-test;Uid=root;Pwd="
+  },
```
Comment (TheDutchDev) [inline AltV.Atlas.Boilerplate.Server/appsettings.json]:
```suggestion
```

Same here, no need for traffic settings. And db module isn't ready yet :P

**Review by TheDutchDev — APPROVED** [review]

**Pull request merged**